src/components/MovieCard/MovieCard.scss

```diff
@@ -0,0 +1,44 @@
+@import "../../common/colors.scss";
+
+.card-item {
+  background: $secondary-color;
+  transition: all 0.3s;
+
+  &:hover {
+    transform: scale(1.1);
+    transition: all 0.3s;
+  }
+}
+
+.card-top {
+  height: 200px;
+
+  img {
+    width: 100%;
+    height: 100%;
+  }
+}
+
+.card-bottom {
+  padding: 20px;
+}
```

Comment:
- [OPTIONAL] Great job so far your mobile Design looks awesome 😉 👍🏻  however I suggest you improve your desktop styling as well to match the original one 👍🏻